control-plane/tests/tests/common/mod.rs

```diff
@@ -211,7 +260,8 @@ impl ClusterBuilder {
             .with_base_image(self.opts.base_image.clone())
             .autorun(false)
             .with_default_tracing()
-            .with_clean(true)
+            // todo: remove before committing
```

Comment:
Do you need to action this todo?

Reply:
that was just a reminder for myself to change the clean back to true again before commiting, which I clearly forgot. I wonder, should we have an ENV variable to tell the composer not to cleanup during local testing? Would avoid having to change those flags all the time while debugging.

Reply:
Done as a separate commit.